src/box/memtx_engine.cc

```diff
@@ -333,7 +346,7 @@ memtx_engine_recover_snapshot_row(struct memtx_engine *memtx,
 	RegionGuard region_guard(&fiber()->gc);
 	if (xrow_decode_dml(row, &request, dml_request_key_map(row->type)) != 0)
 		return -1;
-	*is_space_system = (request.space_id < BOX_SYSTEM_ID_MAX);
+	*is_system_space_row = request.space_id < BOX_SYSTEM_ID_MAX;
```

Comment:
That check was a bit incorrect, it should compare with BOX_SYSTEM_ID_MIN too.

Reply:
Fixed, thanks for noticing!